airflow/www/static/js/gantt-chart-d3v2.js

```diff
@@ -32,11 +32,12 @@ d3.gantt = function() {
     .html(function(d) {
       var s = ""
       s += "<div class='row'>";
-      s += "<div class='col-md-3'>start:<br/>end:<br/>duration:</div>"
+      s += "<div class='col-md-3'>start:<br/>end:<br/>duration:<br/>try_count:</div>"
       s += "<div class='col-md-9'><span style='color: #AAA'> "
       s += d.isoStart + "<br/>";
       s += d.isoEnd + "<br/>";
       s += convertSecsToHumanReadable(d.duration) + "<br/>";
+      s += d.try_count + "<br/>";
```

Comment:
Sry I was misunderstanding this PR. You are right try_number is better.

Reply:
yeah. Making the necessary changes for the same. Thanks for the understanding.